packages/sucrase/src/index.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+
+const { transform } = require('sucrase');
+const { createFilter } = require('rollup-pluginutils');
+
+module.exports = function sucrase(opts = {}) {
+  const filter = createFilter(opts.include, opts.exclude);
+
+  return {
+    name: 'sucrase',
+
+    // eslint-disable-next-line consistent-return
```

Comment:
We could add `return null` at the end of the function instead of suppressing the lint.

Reply:
Let's table this change for now (I'm guilty of this as well) in favor of circling back on the repo to see how many `eslint-disable` statements we can get rid of post-migration.